src/main/scala/gitbucket/core/service/AccountHighlighterService.scala

```diff
@@ -0,0 +1,34 @@
+package gitbucket.core.service
+
+import gitbucket.core.model.AccountHighlighter
+import gitbucket.core.model.Profile._
+import gitbucket.core.model.Profile.profile.blockingApi._
+
+trait AccountHighlighterService {
```

Comment:
I think handling per-user preferences can be parts of `AccountService` because it will highly possibly have further per-user preferences in the future and in that case, adding new service trait per preference item is annoying.

Reply:
@takezoe 
> adding new service trait per preference item is annoying.
Yeah. that make sense. so I will put them into `AccountService`!